### DISCUSSION TITLE
想要二影画的赛斯😭

### DISCUSSION THREAD

**Discussion by YeYingMakas** [share121/inter-knot#5219] [绝区零]
[
![mmexport1726767617774](https://github.com/user-attachments/assets/f1ec11f9-f37b-464a-9ac2-92615362b7a7)
](url)

**Comment by Fluorine-z** [comment]
你不许玩绝区零

**Comment by Anslate** [comment]
你不许玩绝区零

**Comment by Breeze520** [comment]
？你不许玩绝区零

**Comment by wshtdgo** [comment]
？你不许玩绝区零

**Comment by cuijiaqi0406** [comment]
诶🤓👆我149抽也是这两个S😎😎😎
😭🕶👌<img src="https://upload-bbs.mihoyo.com/upload/2022/11/24/3a5cf4c27be061b6b5b8cd0bbf06d54f_267699008387956374.png" alt="蕈兽-晕晕" width="75" >

**Comment by Doubudou45** [comment]
好啊，我用我的赛斯跟你换这个十连怎样～ヽ(*´з｀*)ﾉ

**Comment by Kazuha-hhh** [comment]
破防了

**Comment by d2531987182** [comment]
？你不许玩绝区零
![(F6DQCB%C2V6Z X0((~W)8O](https://github.com/user-attachments/assets/d5af1c53-3bd6-4763-b133-53d057efefd0)

**Comment by summer293** [comment]
（｀Δ´）！你不许玩绝区零

**Comment by nabeilishi** [comment]
爱你！

**Comment by aino2333** [comment]
删了吧，我朋友酸了<img src="https://img-static.mihoyo.com/communityweb/upload/7bd889870907d2d96673d09afb53cf63.png" alt="三月七_哭" width="75" >

**Comment by 210flight** [comment]
你不许玩绝区零
